Lmod has detected the following error: Cannot load module "anaconda/3" because
these module(s) are loaded:
   python

While processing the following module(s):
    Module fullname  Module Filename
    ---------------  ---------------
    anaconda/3       /cvmfs/config.mila.quebec/modules/Core/anaconda/3.lua

[=== Module cudatoolkit/11.7 loaded ===]
[=== Module python/3.10 loaded ===]
Traceback (most recent call last):
  File "/home/mila/s/sonia.joseph/CLIP_AUDIT/clip_audit/sample_images_from_interval.py", line 392, in <module>
    main(args)
  File "/home/mila/s/sonia.joseph/CLIP_AUDIT/clip_audit/sample_images_from_interval.py", line 370, in main
    model = HookedViT.from_pretrained(model_name, is_clip=True, is_timm=False).to('cuda')
  File "/home/mila/s/sonia.joseph/env/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1152, in to
    return self._apply(convert)
  File "/home/mila/s/sonia.joseph/env/lib/python3.10/site-packages/torch/nn/modules/module.py", line 802, in _apply
    module._apply(fn)
  File "/home/mila/s/sonia.joseph/env/lib/python3.10/site-packages/torch/nn/modules/module.py", line 802, in _apply
    module._apply(fn)
  File "/home/mila/s/sonia.joseph/env/lib/python3.10/site-packages/torch/nn/modules/module.py", line 825, in _apply
    param_applied = fn(param)
  File "/home/mila/s/sonia.joseph/env/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1150, in convert
    return t.to(device, dtype if t.is_floating_point() or t.is_complex() else None, non_blocking)
RuntimeError: CUDA error: uncorrectable ECC error encountered
CUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1.
Compile with `TORCH_USE_CUDA_DSA` to enable device-side assertions.

